installation/scripts/install-compass.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+SCRIPTS_DIR="${CURRENT_DIR}/../scripts"
+source $SCRIPTS_DIR/utils.sh
+
+TIMEOUT=30m0s
+
+COMPASS_CHARTS="${CURRENT_DIR}/../../chart/compass"
+
+touch mergedOverrides.yaml # target file where all overrides .yaml files will be merged into. This is needed because if several override files with the same key/s are passed to helm, it applies the value/s from the last file for that key overriding everything else.
+yq eval-all --inplace '. as $item ireduce ({}; . * $item )' mergedOverrides.yaml "${COMPASS_CHARTS}"/values.yaml
+
+POSITIONAL=()
+while [[ $# -gt 0 ]]
+do
+    key="$1"
+
+    case ${key} in
+        --overrides-file)
+            checkInputParameterValue "${2}"
+            yq eval-all --inplace '. as $item ireduce ({}; . * $item )' mergedOverrides.yaml ${2}
+            shift # past argument
+            shift
+        ;;
+        --timeout)
+            checkInputParameterValue "${2}"
+            TIMEOUT="${2}"
+            shift # past argument
+            shift
+        ;;
+        --*)
+            echo "Unknown flag ${1}"
+            exit 1
+        ;;
+        *)    # unknown option
+            POSITIONAL+=("$1") # save it in an array for later
+            shift # past argument
+        ;;
+    esac
+done
+set -- "${POSITIONAL[@]}" # restore positional parameters
+
+CRDS_FOLDER="${CURRENT_DIR}/../resources/crds"
+kubectl apply -f "${CRDS_FOLDER}"
+helm upgrade --install --wait --timeout "${TIMEOUT}" -f ./mergedOverrides.yaml --create-namespace --namespace compass-system compass "${COMPASS_CHARTS}"
+rm mergedOverrides.yaml
```

Comment:
If something fail the file will be left behind.

Add it to gitignore for such cases.

Also you can use trap to minimize such cases: https://github.com/kyma-incubator/compass/blob/e495587b9928d436701650783fb9e453d03b424f/installation/cmd/run.sh#L184